tsl/test/shared/sql/git_stuff.sql

```diff
@@ -0,0 +1,10 @@
+-- This file and its contents are licensed under the Timescale License.
```

Comment:
I am not too fond of the name "git_stuff". Maybe just "build_info" or something?